source/code-generation/download-all-code-files.txt

```diff
@@ -1,13 +1,45 @@
+.. _rm-download-all-code-files:
+
 =======================
 Download All Code Files
 =======================
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-TBD.
+Code Generation is a feature built into Relational Migrator that helps
+automatically generate code files for your applications connected 
+to MongoDB. This page details how to generate and download all code files into a zip file.
+
+About this Task
+---------------
+
+Some templates also generate additional files which are not specific to 
+a MongoDB collection. These files are called **Support files**. 
+
+Click the **Support files** check box to indicate if you would like 
+these files included in your downloads.
+
+Steps
+-----
+
+Navigate to the **Code Generation** screen, in the left **Generate code** pane:
+
```

Comment:
(format)
I do recommend that we use the procedure directive for task pages.

Reply:
Discussed, possible re-write for future, but because of the excessive space the procedure directive takes up this will remain a numbered list for now.